dbms/src/AggregateFunctions/AggregateFunctionNull.h

```diff
@@ -505,34 +506,16 @@ class AggregateFunctionNullVariadic final
 };
 
 // Make the prefix_size >= sizeof(UInt64) and could fit the align size
-inline size_t enlarge_prefix_size(size_t prefix_size) noexcept
+inline size_t enlargePrefixSize(size_t prefix_size) noexcept
 {
     assert(prefix_size != 0);
+    static_assert((sizeof(prefix_size_look_up_table) / sizeof(UInt64)) == 7);
```

Comment:
asdd another static_assert to make sure sizeof(UInt64) == 8?

Reply:
> asdd another static_assert to make sure sizeof(UInt64) == 8?

done